versioned_docs/version-1.0/json-cadence-spec.md

```diff
@@ -750,22 +766,22 @@ Initializer types are encoded a list of parameters to the initializer.
   "kind": "Restriction",
```

Comment:
This no longer exists in Cadence 1.0

Reply:
I've updated to say `Intersection`, but after seeing some of the tests, im not confident this is a good example.

However, this "restricted" bug in the doc seems to be carried over from the migration, @dsainati1 did you update this section in relation to your FLIP last?

https://github.com/onflow/flips/pull/85
https://github.com/onflow/cadence/issues/2267#issuecomment-1673272779

Reply:
https://cadence-lang.org/docs/1.0/language/intersection-types this should be the new link